R/geom-violin.r

```diff
@@ -107,7 +110,19 @@ GeomViolin <- proto2(
       # Needed for coord_polar and such
       newdata <- rbind(newdata, newdata[1,])
 
-      ggname(self$my_name(), GeomPolygon$draw(newdata, ...))
+      if (any(data$is.quantile)) {
```

Comment:
I think these should be a parameter of the draw function, rather than an aesthetic.